routing/adjust_fees.js

```diff
@@ -308,10 +315,14 @@ module.exports = (args, cbk) => {
             // Only the highest fee rate across all peer channels applies
             const maxFeeRate = max(...currentPolicies.map(n => n.fee_rate));
 
+            // Only the least inbound discount rate across all peer channels applies
+            const maxInboundDiscountRate = min(...currentPolicies.map(n => n.inbound_rate_discount));
```

Comment:
should this be const minInboundDiscountRate since only the smallest discount applies in pathfinding?

Reply:
fixed it.